yarn.lock

```diff
@@ -10275,6 +10295,11 @@ simple-fmt@~0.1.0:
   version "0.1.0"
   resolved "https://registry.yarnpkg.com/simple-fmt/-/simple-fmt-0.1.0.tgz#191bf566a59e6530482cb25ab53b4a8dc85c3a6b"
 
+simple-guid@0.0.1:
+  version "0.0.1"
+  resolved "https://registry.yarnpkg.com/simple-guid/-/simple-guid-0.0.1.tgz#d5e819f3c7f7ad3d1a8821eab60d8fed0d3917ea"
+  integrity sha1-1egZ88f3rT0aiCHqtg2P7Q05F+o=
+
```

Comment:
Looks like this PR, https://github.com/department-of-veterans-affairs/vets-website/pull/9763, forgot to run `yarn install`.